entry_types/scrolled/package/src/frontend/PlayerControls/PlayPauseButton.js

```diff
@@ -29,6 +32,10 @@ function clickHandler(props) {
       props.pause();
     } else {
       props.play();
+      if (!isBlessed) {
+        media.mute(false);
+        isBlessed = true;
+      }
     }
 
```

Comment:
`PlayerControls` is a presentational/dumb component that so far does not know about concepts like media. There is also `MediaPlayerControls`, which wires up the `PlayerControls` with `playerState`/`playerActions`. I think, it would be a better fit for media related functionality. Can you move this to a separate PR so it can already be merged?

Reply:
I thought this player actions.play might get invoked from other places. I have rebased this PR can be merged.